main/webapp/modules/core/scripts/index/open-project-ui.js

```diff
@@ -101,15 +101,54 @@ Refine.OpenProjectUI.prototype._fetchProjects = function() {
 };
 
 Refine.OpenProjectUI.prototype._buildTagsAndFetchProjects = function() {
+    this._buildProjectSearchPanel();
     this._buildTagsListPanel();
     this._fetchProjects();
 };
 
+Refine.OpenProjectUI.prototype._buildProjectSearchPanel = function(){
+    var self = this;
+    self._allTags = Refine.TagsManager._getAllProjectTags();
+    var container = self._elmts.projectTags.empty();
+    // Add search menu item
+    var div = $('<div/>')
+    .attr('id','divSearch')
+    .appendTo(container)
+    // Add form to the div on the left
+    var form = $('<div/>')
+    .attr('id','formSearch')
+    .appendTo(div);
+    // Add div for image in the form
+    var divImage = $('<div/>')
+    .attr('id','divImage')
+    .appendTo(form)
+    // Add img to the form
+    $('<div/>')
+    .html('<svg xmlns="http://www.w3.org/2000/svg" height="100%" viewBox="0 0 24 24" width="100%"><path d="M0 0h24v24H0z" fill="none"/><path d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"/></svg>')
+    .attr('id', 'searchIcon')
+    .addClass("magnifying_glass")
+    .appendTo(divImage);
+
+    // Add div for input, in the form
```

Comment:
Oops!  Looks like we are losing the highlighting of the Tag Bubble's after I unclick the Search magnifying glass... however the Tag Bubble highlighting works the first time.  So your Search Input box slide out, might somehow be destroying the Tag Bubble highlighting?

Reply:
@thadguidry I haven't be able to reproduce this, can you give me more info?
I tried this
![Screen Recording 2020-12-14 at 11 27 56](https://user-images.githubusercontent.com/16580567/102114288-a3dc2300-3dff-11eb-8d5f-5beb511d42bb.gif)

Reply:
@mayralgr I cannot reproduce now either.  Might have been temporary issue with Firefox and a breakpoint I had not turned off.
Sorry!
This looks good now.
@wetneb I'd say ready for merge.